krun.py

```diff
@@ -330,8 +331,4 @@ def attach_log_file(config, resume):
     debug("arguments: %s"  % " ".join(sys.argv[1:]))
     parser = create_arg_parser()
     setup_logging(parser)
-
-    try:
-        main(parser)
-    except:
-        sys.exit(1)
+    main(parser)
```

Comment:
Yes, I think so. What we've done is sensible but it won't be obvious how the exceptions flow for a first-time reader of this code (or me in five minutes time)